src/extras.jl

```diff
@@ -60,17 +61,21 @@ i.e. the lower bound is included and the upper bound is excluded.
 If `x` accepts missing values (i.e. `eltype(x) >: Missing`) the returned array will
 also accept them.
 
-# Arguments
+# Keyword arguments
 * `extend::Bool=false`: when `false`, an error is raised if some values in `x` fall
   outside of the breaks; when `true`, breaks are automatically added to include all
   values in `x`, and the upper bound is included in the last interval.
 * `labels::Union{AbstractVector,Function}: a vector of strings giving the names to use for
-  the intervals; or a function `f(from, to, i; closed)` that generates the labels from the
-  left and right interval boundaries and the group index. Defaults to 
+  the intervals; or a function `f(from, to, i; leftclosed, rightclosed)` that generates
+  the labels from the left and right interval boundaries and the group index. Defaults to
   `"[from, to)"` (or `"[from, to]"` for the rightmost interval if `extend == true`).
 * `allow_missing::Bool=true`: when `true`, values outside of breaks result in missing values.
   only supported when `x` accepts missing values.
-  
+* `allowempty::Bool=false`: when `false`, an error is raised if some breaks appear
```

Comment:
This is OK. Probably `allowempty=true` here is not very intuitive, but sometimes breaks might be generated programmatically so it is OK to have this.